app/views/content/landing/grow/_collage.html.erb

```diff
@@ -0,0 +1,20 @@
+<div class="row">
+  <div class="grey col">
+    <section class="col col-full-content">
+      <div class="inset">
+        <p class="col-720">Teaching is a fulfilling career. Not only can you inspire the pupils you teach, you also get to learn new skills, build your confidence and become your best self. The benefits of being a teacher include:</p>
+        <%= render Content::ChecklistCollageComponent.new(
+          checklist: [
+            "career progression ",
+            "personal development",
+            "opportunities to increase your pay"
+          ],
+          image_paths: [
+            "static/content/grow/Smiling_Student.jpg",
+          ]
+        ) do %>
```

Comment:
there is no need for an empty

```
do
end
```

block. Instead, change line 15 to:
```
     )  %>
```
and delete line 16 (` <% end %>`)

Reply:
@garethwilliamsdesign @martyn-w Could you resolve this between you? We need to get this published today. Thanks!

Reply:
@gemmadallmandfe @martyn-w @garethwilliamsdesign think I've updated it correctly if someone can double check. Then it should be ready to approve. Thanks